proposed/clock-meta.md

```diff
@@ -2,21 +2,19 @@
 
 ## 1. Summary
 
-The purpose of using the `ClockInterface` is to provide a standard way to access the system 
-time, that would allow interopability when testing code that relies on the current time 
-rather than relying on installing PHP extensions or use hacks like re-declaring the `time()`
-function in other namespaces. 
+Getting the current time in applications is a rather straightforward thing given the language's native `time()` function (or one of its similar functionalities like `utime()` or `new DateTimeImmutable()` etc.)
```

Comment:
s/utime/microtime I suppose?

Reply:
Good catch! Thanks. Mixed up the underlying C-functions there 😁